.github/workflows/release.yml

```
@@ -0,0 +1,58 @@
+name: Docfx V3 Release
+on:
+  pull_request:
+    branches: [ v3 ] # for testing
+jobs:
+  build:
+    name: Release
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-dotnet@v1
+    - name: Donet pack
+      # if: ${{ env.BUILD_REASON != 'pull_request' }}
+      shell: pwsh
+      run: |
```

Remove the `Remove-Item` as we are on a hosted machine.
```suggestion
      run: dotnet pack -c Release -o drop
```